core/templates/pages/exploration-player-page/services/exploration-player-state.service.spec.ts

```diff
@@ -134,8 +133,6 @@ describe('Exploration Player State Service', () => {
     StatsReportingService = _StatsReportingService_;
     PlaythroughIssuesService = _PlaythroughIssuesService_;
     PlaythroughService = _PlaythroughService_;
-    ReadOnlyExplorationBackendApiService = (
```

Comment:
Is this (code below) still relevant after this change?

https://github.com/oppia/oppia/blob/b1ba8d11e81b968cddaaa66180f8b787e7c72372/core/templates/pages/exploration-player-page/services/exploration-player-state.service.spec.ts#L74-L84

Reply:
Done.